### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by xmpf**
README.md: Changed the example code, to support custom installation paths.

osquery/management.py: Replaced (is not 0) with ( != 0 ).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,15 +91,22 @@ Extensions are the core way that you can extend and customize osquery. At Facebo
...
 
 ```python
+#!/usr/bin/env python3
+
 import osquery
+import pprint
+import osquery.management as management
+
+# Override default location: /usr/bin/osqueryd
+management.LINUX_BINARY_PATH = "/usr/local/bin/osqueryd"
```
Comment (theopolis) [inline README.md:101]: I think `/usr/bin/osqueryd` is the default if you install osquery through a package and `/usr/local/bin/osqueryd` is the default if you use `make install` (or through the hosted tgz bundle).

My recommendation is to make the change in the example, but rather keep this line commented-out for people who install via `make install`.

File: README.md [hunk 2]
```diff
@@ -91,15 +91,22 @@ Extensions are the core way that you can extend and customize osquery. At Facebo
...
 
 if __name__ == "__main__":
     # Spawn an osquery process using an ephemeral extension socket.
     instance = osquery.SpawnInstance()
     instance.open()  # This may raise an exception
 
     # Issues queries and call osquery Thrift APIs.
-    instance.client.query("select timestamp from time")
+    pprint.pprint(instance.client.query("select * from hosts").response)
```
Comment (theopolis) [inline README.md:109]: Do you mind breaking this out into two distinct lines? One line to store the result from the API call and another to demonstrate that you can pretty-print the response member?

**Review by theopolis — CHANGES_REQUESTED** [review]
Thanks for the PR, I have a few recommendations, let me know what you think.

File: README.md (hunk 2 above)
Comment (xmpf) [reply README.md]: Yeah sure and sorry for the double commit :)

File: README.md (hunk 1 above)
Comment (xmpf) [reply README.md]: It's also possible to define the path, as a parameter to the `SpawnInstance()` method - thus I will close this PR.

**Pull request closed without merging**